### DISCUSSION TITLE
MissionControl 0.10.0

### DISCUSSION THREAD

**Discussion by ndeadly** [ndeadly/MissionControl#735] [General]
<p align="left">
<img alt="GitHub Releases" src="https://img.shields.io/github/downloads/ndeadly/MissionControl/v0.10.0/total">
<a href="https://discord.gg/gegfNZ5Ucz"><img alt="Discord Server" src="https://discordapp.com/api/guilds/905069757361971303/widget.png?style=shield"/></a>
</p>

This is the twenty-second official release of Mission Control.

This release adds support for firmware 17.0.0, a new controller, and a few bugfixes and improvements. See below for full changelog.

New users landing on this release page should first check out the readme on the [main project page](https://github.com/ndeadly/MissionControl#readme) for the official project documentation. There you can find installation and usage instructions along with an FAQ section that will answer most of your questions.

_PS: there is some ongoing testing of preliminary Bluetooth LE support happening in my discord server (see #testing). If you own any of the controllers listed [here](https://github.com/ndeadly/MissionControl/issues?q=is%3Aissue+is%3Aopen+label%3A%22BLE+controller%22) or any LE controllers that are not on the list, we'd like to hear from you! (though I don't need any additional input on Xbox and Stadia controllers for now). Please note that these are alpha builds for the purpose of gathering controller information and furthering my understanding of how BLE works on the console. They are **_NOT_** intended for general use. There _will_ be bugs and issues. There are no ETAs. I will not be providing any support - if you can't follow the usage instructions, then this is not for you._

### Changelog:
- Added full support for firmware 17.0.0
- Added support for Atari Modern Wireless Controller
- Mapped 8BitDo Zero D-pad as left analog stick to improve game compatibility
- Removed unnecessary gyro scaling factor for Sony controllers. Controller rotations should now be more sensitive/accurate.
- Fixed an issue where capture button state wasn't being reset after Dualshock 4 touchpad button released.

<hr /><em>This discussion was created from the release <a href='https://github.com/ndeadly/MissionControl/releases/tag/v0.10.0'>MissionControl 0.10.0</a>.</em>

**Comment by LuisTesta86** [comment]
Hello, is it compatible with version 17.0.1?

**Reply by ndeadly** [reply]
Yes. If a new firmware doesn't require a new atmosphere release, it almost certainly won't require a mission control update either.

**Comment by nailaojunC** [comment]
Unable to connect to Bluetooth after updating 17.0.1

**Reply by ndeadly** [reply]
Nothing has changed in the latest firmware.

**Reply by darkwolf2212** [reply]
Friend, did you find the solution?????

**Comment by Vellmaxi** [comment]
BLE support for year 2024 Please T.T
Where is your discord links ?

**Reply by ndeadly** [reply]
We'll see, fingers crossed. Discord links can be found at the top of the main project readme or release notes on github.

**Reply by valenparra** [reply]
Hello ndeadly,

A big thank you for your releases, this is one of the must have tools for Atmosphere! I have updated my firmware to 17.0.1 and made a new clean install of my whole SD card. I have tested this MissionControl version and unfortunately it's not working for me (I tested WiiU pro and Dualshock 4 V2 gamepads). These 2 gamepads worked perfectly in other releases I have used, so maybe I am doing something wrong this time. Am I supposed to reboot my console or other thing to make it work? I just dragged and dropped the files into the SD card root, as always, but the gamepads are not being recognized to be paired.

Thanks in advance for your help!

**Reply by ndeadly** [reply]
Yes, you need to reboot whenever you install a new version

**Comment by darkwolf2212** [comment]
Hello, how are you? I already installed it but my firmware is 17.0.1 and unfortunately it doesn't work for me. What I can do ???

**Reply by ndeadly** [reply]
English only, please

**Reply by darkwolf2212** [reply]
Hello, now it is already in English, thank you

**Comment by XianRise** [comment]
After updating to 17.0.1, Bluetooth cannot be connected normally after dragging files to the SD card. Please resolve.

**Comment by ActiveSierro** [comment]
when i play smash bros idk why but my friends like to use the ps4 and ps5 whenever I disconnect the controller in smash bros and start the game it will crash only when i disconnect it
